common-test/src/main/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -203,7 +203,7 @@ public ProcessResult exec(String pod, String... command) {
     }
 
     private K waitFor(String resource, String name, Predicate<JsonNode> ready) {
-        long timeoutMs = 600_000L;
+        long timeoutMs = 570_000L;
```

Comment:
Just out of curiosity ... why 570_000? Is that a test whether we do a thorough review? :-)

Reply:
Travis times out after 10 minutes of no output. It slightly more helpful for us to timeout first (since that way we get diagnostic info about why it was taking so long) since travis will just kill the build with no diagnostics otherwise.

Reply:
Yeah, that makes sense. Thanks for explaining.